packages/react/src/drafts/IssueLabel/getColorFromHex.ts

```diff
@@ -0,0 +1,137 @@
+import {getContrast} from 'color2k'
+import {Hsluv} from 'hsluv'
+
+type Hex = `#${string}`
```

Comment:
One idea I had for this was to pull this out into a dedicated file, maybe a `types.ts` or `color.ts` file in the folder, and export it so that we can use it in this file, in stories, and in the component without having to re-type out `#${string}` in make TS happy